data/wacom-hid-52EB-pen.tablet

```diff
@@ -0,0 +1,22 @@
+# Wacom
+# Wacom HID 52EB Pen
+# WACF2200
+#
+# sysinfo.0VeQaUVnQ1
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/369
+
+[Device]
+Name=Wacom HID 52EB Pen, Wacom HID 52EB Finger
```

Comment:
```suggestion
Name=Wacom HID 52EB Pen
```
The finger device is the touch device and doesn't have any effect on libwacom so better to leave it out